napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -0,0 +1,88 @@
+import sys
+from typing import List
+
+from app_model.types import Action, KeyCode, KeyMod, StandardKeyBinding
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....settings import get_settings
+from ....utils.translations import trans
+from ...qt_main_window import Window
+from ...qt_viewer import QtViewer
+
+
+def _toggle_activity_dock(window: Window):
+    window._status_bar._toggle_activity_dock()
+
+
+Q_VIEW_ACTIONS: List[Action] = [
+    Action(
+        id=CommandId.TOGGLE_FULLSCREEN,
+        title=CommandId.TOGGLE_FULLSCREEN.title,
+        menus=[
+            {
+                'id': MenuId.MENUBAR_VIEW,
+                'group': MenuGroup.NAVIGATION,
+                'order': 1,
+            }
+        ],
+        callback=Window._toggle_fullscreen,
+        keybindings=[StandardKeyBinding.FullScreen],
+    ),
+    Action(
+        id=CommandId.TOGGLE_MENUBAR,
+        title=CommandId.TOGGLE_MENUBAR.title,
+        menus=[
+            {
+                'id': MenuId.MENUBAR_VIEW,
+                'group': MenuGroup.NAVIGATION,
+                'order': 2,
+                'when': sys.platform != 'darwin',
+            }
+        ],
+        callback=Window._toggle_menubar_visible,
+        keybindings=[
+            {
+                'win': KeyMod.CtrlCmd | KeyCode.KeyM,
+                'linux': KeyMod.CtrlCmd | KeyCode.KeyM,
+            }
+        ],
+        # TODO: add is_mac global context keys (rather than boolean here)
+        enablement=sys.platform != 'darwin',
+        status_tip=trans._('Show/Hide Menubar'),
+    ),
+    Action(
+        id=CommandId.TOGGLE_PLAY,
+        title=CommandId.TOGGLE_PLAY.title,
+        menus=[
+            {
+                'id': MenuId.MENUBAR_VIEW,
+                'group': MenuGroup.NAVIGATION,
+                'order': 3,
+            }
+        ],
+        callback=Window._toggle_play,
+        keybindings=[{'primary': KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyP}],
+    ),
+    Action(
+        id=CommandId.TOGGLE_OCTREE_CHUNK_OUTLINES,
+        title=CommandId.TOGGLE_OCTREE_CHUNK_OUTLINES.title,
+        menus=[
+            {
+                'id': MenuId.MENUBAR_VIEW,
+                'group': '1_render',
+                'order': 1,
+                'when': get_settings().experimental.octree,
+            }
+        ],
+        callback=QtViewer._toggle_chunk_outlines,
+        enablement=get_settings().experimental.octree,
+        # this used to have a keybinding of Ctrl+Alt+O, but that conflicts with
+        # Open files as stack
+    ),
+    Action(
+        id=CommandId.TOGGLE_ACTIVITY_DOCK,
+        title=CommandId.TOGGLE_ACTIVITY_DOCK.title,
+        menus=[{'id': MenuId.MENUBAR_VIEW, 'group': '1_render', 'order': 11}],
```

Comment:
Where is the set of possible `group` entries defined? I can see that a good number come from the `_app_model` constants, but I also see `1_render` as a string? Why is the `render` group not also defined n the `constants` file?

Reply:
should be!

Reply:
Has this been resolved? AFAICT the constants file has:
```python
__all__ = ['CommandId', 'MenuGroup', 'MenuId']
```

Reply:
Amended in https://github.com/napari/napari/pull/4826/commits/47cef11c55868f9ebcc04fc22860481cc6abb736, let me know if it is not correct.